app/frontend/src/features/communities/events/CreateEventPage.tsx

```diff
@@ -0,0 +1,95 @@
+import { Error as GrpcError } from "grpc-web";
+import { Event } from "proto/events_pb";
+import { communityEventsBaseKey } from "queryKeys";
+import { useMutation, useQueryClient } from "react-query";
+import { service } from "service";
+import type { CreateEventInput } from "service/events";
+import dayjs from "utils/dayjs";
+import makeStyles from "utils/makeStyles";
+
+import { CREATE_EVENT, CREATE_EVENT_SUCCESS } from "./constants";
+import EventForm, { CreateEventData } from "./EventForm";
+
+const useStyles = makeStyles((theme) => ({}));
+
+export default function CreateEventPage() {
+  const queryClient = useQueryClient();
+  const {
+    mutate: createEvent,
+    error,
+    isLoading,
+    isSuccess,
+  } = useMutation<
+    Event.AsObject,
+    GrpcError,
+    CreateEventData,
+    { parentCommunityId?: number }
+  >(
+    (data) => {
+      let createEventInput: CreateEventInput;
+      const finalStartDate = data.startDate
+        .startOf("day")
+        .add(dayjs(data.startTime).get("hour"), "hour")
+        .add(dayjs(data.startTime).get("minute"), "minute")
+        .toDate();
+      const finalEndDate = data.endDate
+        .startOf("day")
+        .add(dayjs(data.endTime).get("hour"), "hour")
+        .add(dayjs(data.endTime).get("minute"), "minute")
+        .toDate();
+
+      if (data.isOnline) {
+        createEventInput = {
+          isOnline: data.isOnline,
+          title: data.title,
+          content: data.content,
+          photoKey: data.eventImage,
+          startTime: finalStartDate,
+          endTime: finalEndDate,
+          // TODO: not hardcode this and allow user to specify community ID?
+          parentCommunityId: 1,
```

Comment:
Until we do this or decide on something, I think add a little subtitle to the checkbox "Virtual events will be added to the global community"